code/game/objects/items/handcuffs.dm

```diff
@@ -7,10 +7,17 @@
  * 1. A special suicide
  * 2. If a restraint is handcuffing/legcuffing a carbon while being deleted, it will remove the handcuff/legcuff status.
 */
+
+// Zipties, cable cuffs, etc. Can be cut with wirecutters instantly.
+#define HANDCUFFS_TYPE_WEAK 0
+// Handcuffs... alien handcuffs. Can be cut through only by jaws of life.
+#define HANDCUFFS_TYPE_STRONG 1
+
 /obj/item/restraints
 	breakouttime = 1 MINUTES
 	dye_color = DYE_PRISONER
 	icon = 'icons/obj/restraints.dmi'
+	var/restraint_strength = HANDCUFFS_TYPE_WEAK
```

Comment:
why have it on restraints if it doesnt work for legcuffs?

Reply:
i was planning on doing it but it would be too much of a balance change so i refrained for now, could be an interesting bola counterplay tho